src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -2551,6 +2556,10 @@ private void processFileChanges(
                                                                                             // files
             if (fileChanged.exists() && (changeType == ChangeType.MODIFY
                     || changeType == ChangeType.CREATE)) {
+                if (fileChanged.getName().equals("server.xml")) {
```

Comment:
This could also be other xml config files (e.g. xml files that are included by server.xml) which can trigger install-feature too.  Perhaps check if the filename ends with `.xml`, so that adding features in those other files will also skip the warning.

Or, is there any need to check the filename at all?  Even if it's some other non-xml file (e.g. bootstrap.properties), would it be fine to set `skipBetaInstallFeatureWarning` to `true`, and let it be reset to `false` afterwards (which it seems to already do)?